core-common/essentials-deploy/deploy.bat

```diff
@@ -0,0 +1,56 @@
+@echo OFF
```

Comment:
A Linux-version would be good too.

Reply:
I've put a TODO remark into the batch script